docs/Manual/03 Schedules.md

```diff
@@ -62,10 +63,11 @@ for k in range(11):
             C[i, j] += A[i, k] * B[k, j]
 ```
 
-Some orders are not allowed. Describing the restrictions in full requires concepts that have not yet been introduced, so we merely mention the restrictions here and explain them in detail later on. The restrictions are:
-1. The *inner dimension* created by a `split` transformation (see below) always comes after its corresponding *outer dimension*.
+However, some orders are not allowed. Describing these restrictions in full will require concepts that are yet to be introduced. Therefore, we are stating these restrictions here and will discuss them later in the upcoming sections. The restrictions are: 
+1. The *inner dimension* created by a `split` transformation (see below) is always followed by its corresponding *outer dimension*.
```

Comment:
`The *inner dimension* created by a `split` transformation (see below) must be ordered after its corresponding *outer dimension*.`

(i.e. not followed by, the outer dimension must be ordered before the inner dimension)

Reply:
I think the use of after / later than is still a bit confusing, but it's ok. We can leave it for now, the subject is the dimension created by split or fusing and how they should be ordered relative to the other indices.

Reply:
how about we replace "later than" with "after"?

Reply:
"after" seems more appropriate.